docs/source/app_developers_guide/about_events.rst

```diff
@@ -0,0 +1,123 @@
+*********************
+About Sawtooth Events
+*********************
+
+Sawtooth events occur "on block boundaries" - that is, when blocks are
```

Comment:
Putting quotes around ""on block boundaries"" seems makes it seem like you are being intentionally vague. If that language is vague or confusing, just say "Sawtooth events occur when blocks are committed". Also, "occur" feels like the wrong word here. I prefer "are broadcasted" or something similar to indicate that this is when you will receive the event, not necessarily when it happened, although they are usually the same or closely linked.